addon/components/link-to-external-component.js

```diff
@@ -9,9 +9,11 @@ export default LinkComponent.extend({
     const owner = getOwner(this);
 
     if (owner.mountPoint) {
-      const targetRouteName = get(this, 'targetRouteName');
-      const externalRoute = owner._getExternalRoute(targetRouteName);
-      set(this, 'targetRouteName', externalRoute);
+      // https://emberjs.github.io/rfcs/0459-angle-bracket-built-in-components.html
```

Comment:
can you add tests for this, please?

Reply:
We can't really specifically test for this, since this is dependent on the Ember version. However, `{{link-to}}` and `{{link-to-external}}` are tested extensively in the acceptance tests. I have verified, that in the `ember-beta` scenario without my fixes 25 tests are failing, while with my fixes applied only 1 unrelated test fails.

I plan to fix that test in a separate PR.

Reply:
I think that you can refactor this test with angle brackets - https://github.com/ember-engines/ember-engines/blob/master/tests/integration/component-with-link-to-external-test.js#L21-L26

since that your PR propose to do it, adds support to angle-bracket invocation

Reply:
Added angle bracket invocation tests for Ember `3.10.0-beta.1` and upwards.